### PR TITLE
Tell users to build with ZOLTAN.

### PR DESCRIPTION
**Pull request by bangerth**
Follow-up to #5514.

### CODE REVIEW

File: doc/external-libs/trilinos.html [hunk 1]
```diff
@@ -87,6 +88,7 @@ <h5>Installing Trilinos</h5>
     -DTrilinos_ENABLE_MueLu=ON                       \
```
Comment (tjhei) [inline doc/external-libs/trilinos.html]: can you update the path in line 77 to "-12.10.1"?

File: doc/external-libs/trilinos.html [hunk 2]
```diff
@@ -87,6 +88,7 @@ <h5>Installing Trilinos</h5>
...
     -DTrilinos_ENABLE_ML=ON                          \
     -DTrilinos_ENABLE_ROL=ON                         \
+    -DTrilinos_ENABLE_Zoltan=ON                      \
```
Comment (tjhei) [inline doc/external-libs/trilinos.html]: is this really needed? We do not specify it in https://github.com/dealii/candi/blob/master/deal.II-toolchain/packages/trilinos.package#L146 for example but it certainly builds zoltan.

Comment (bangerth) [reply doc/external-libs/trilinos.html]: Are you saying that with the existing flags, the Trilinos dependency tracking determines that it needs to enable Zoltan anyway? Either way, I think we all just copy-paste these flags, so why not err on the safe side and ask users to enable it explicitly.

File: doc/external-libs/trilinos.html (hunk 1 above)
Comment (bangerth) [reply doc/external-libs/trilinos.html]: I did something similar.

File: doc/external-libs/trilinos.html (hunk 2 above)
Comment (davydden) [reply doc/external-libs/trilinos.html]: i would keep it, it can't harm

Comment (tjhei) [reply doc/external-libs/trilinos.html]: > Are you saying that with the existing flags, the Trilinos dependency tracking determines that it needs to enable Zoltan anyway?

Yes.

**Comment by tamiko** [comment]
@tjhei I don't see a problem in explicitly enabling zoltan either. Shall we merge?

**Comment by masterleinad** [comment]
@tjhei Do you have strong objections against the current version?

**Pull request merged**